sdk/tables/azure-data-tables/azure/data/tables/aio/_table_service_client_async.py

```diff
@@ -0,0 +1,307 @@
+        """Sets properties for an account's Table service endpoint,
+        including properties for Analytics and CORS (Cross-Origin Resource Sharing) rules.
+
+
+        :param cors:
+        :param minute_metrics:
+        :param hour_metrics:
+        :param analytics_logging:
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: None, or the result of cls(response)
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        props = TableServiceProperties(
+            logging=analytics_logging,
+            hour_metrics=hour_metrics,
+            minute_metrics=minute_metrics,
+            cors=cors
+        )
+        try:
+            return await self._client.service.set_properties(props, **kwargs)  # type: ignore
+        except HttpResponseError as error:
+            process_table_error(error)
+
+    @distributed_trace_async
+    async def create_table(
+            self,
+            table_name,
+            headers=None,
+            **kwargs
+    ):
+        # type: (...) -> TableClient
+        """Creates a new table under the given account.
+
+        :param headers:
+        :param table_name: The Table name.
+        :type table_name: ~azure.data.tables._models.Table
+        :return: TableClient, or the result of cls(response)
+        :rtype: ~azure.data.tables.TableClient or None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        table_properties = TableProperties(table_name=table_name, **dict(kwargs, headers=headers))
+        table = self.get_table_client(table=table_name)
+        await self._client.table.create(table_properties=table_properties, **kwargs)
+        return table
+
+    @distributed_trace_async
+    async def delete_table(
+            self,
+            table_name,
+            request_id_parameter=None,
+            **kwargs
+    ):
+        # type: (...) -> None
+        """Creates a new table under the given account.
+
+                        :param request_id_parameter: Request Id parameter
+                        :type request_id_parameter: str
+                        :param table_name: The Table name.
+                        :type table_name: str
+                        :keyword callable cls: A custom type or function that will be passed the direct response
+                        :return: None
+                        :rtype: ~None
+                        """
+        response = await self._client.table.delete(table=table_name, request_id_parameter=request_id_parameter,
+                                                   **kwargs)
+        return response
+        # table = self.get_table_client(table=table_name)
+        # table.delete_queue(table_name)
+
+    @distributed_trace
+    def list_tables(
+            self,
+            query_options=None,  # type: Optional[QueryOptions]
+            **kwargs
+    ):
+        # type: (...) -> AsyncItemPaged
```

Comment:
this shouldnt take in a queryOptions 

select, results_per_page and filter should be keyword args